components/clarinet-sdk/tests/reports.test.ts

```diff
@@ -0,0 +1,109 @@
+import fs from "node:fs";
+import path from "node:path";
+import { describe, expect, it, beforeEach, afterEach } from "vitest";
+
+// test the built package and not the source code
+// makes it simpler to handle wasm build
+import { Simnet, initSimnet } from "../dist/esm";
+
+const address1 = "ST1SJ3DTE5DN7X54YDH5D64R3BCB6A2AG2ZQ8YPD5";
+
+const deploymentPlanPath = path.join(
+  process.cwd(),
+  "tests/fixtures/deployments/default.simnet-plan.yaml",
+);
+
+function deleteExistingDeploymentPlan() {
+  if (fs.existsSync(deploymentPlanPath)) {
+    fs.unlinkSync(deploymentPlanPath);
+  }
+}
+
+beforeEach(async () => {
+  deleteExistingDeploymentPlan();
+});
+
+afterEach(() => {
+  deleteExistingDeploymentPlan();
+});
```

Comment:
Do we need to do this before and after?

Reply:
`beforeEach` is definitely required to make sure the first test is properly ran.
The  `afterEach` _could_ be replaced by an `afterAll` just to clean the project after the test ran (or we could not clean it at all

Reply:
👍